Mathlib/Algebra/Category/Ring/Colimits.lean

```diff
@@ -12,9 +12,9 @@ import Mathlib.CategoryTheory.ConcreteCategory.Elementwise
 /-!
 # The category of commutative rings has all colimits.
 
-This file uses a "pre-automated" approach, just as for `Mon/colimits.lean`.
+This file uses a "pre-automated" approach, just as for `MonCat/Colimits.lean`.
```

Comment:
Can you change this to use the full path? It will be easier to teach doc-gen how to link those.